code/game/objects/items/weapons/twohanded.dm

```diff
@@ -377,6 +377,37 @@
 		desc = "A makeshift spear with [C42] attached to it. Alt+click on the spear to set your war cry!"
 	update_icon()
 
+
+//GREY TIDE
+
+/obj/item/weapon/twohanded/spear/grey_tide
+	icon_state = "spearglass0"
+	name = "\improper Grey Tide"
+	desc = "Recovered from the aftermath of a revolt aboard Defense Outpost Theta Aegis, in which a seemingly endless tide of Assistants caused heavy casualities among Nanotrasen military forces. Alt+click to bind it to yourself."
+	force_unwielded = 15
+	force_wielded = 25
+	throwforce = 20
+	throw_speed = 4
+	attack_verb = list("gored")
+	var/mob/living/spear_owner
+
+/obj/item/weapon/twohanded/spear/grey_tide/afterattack(atom/movable/AM, mob/living/user, proximity)
+	..()
+	if(spear_owner != user)
+		user.faction = "greytide([rand(1, 1000)])"
+		spear_owner = user
+	if(istype(AM, /mob/living))
+		var/mob/living/L = AM
+		if(!L.stat && prob(50))
+			var/mob/living/simple_animal/hostile/illusion/M = new(user.loc)
+			M.faction = user.faction
+			M.Copy_Parent(user, 100, user.health/2.5, 12, 30)
+			M.GiveTarget(L)
+
+
+/obj/item/weapon/twohanded/spear/AltClick()
+
+

```

Comment:
Oh whoops, I was planning on letting you bind it with alt+click but just made it automated instead